videos-list/05-dario-hessam.md

```diff
@@ -13,12 +13,25 @@ Discourse Discussion
 https://discourse.pymc.io/t/an-alcohol-what-are-the-chances-knowledge-based-and-probabilistic-models-in-chemistry-using-pymc3-by-hessam-mehr-dario-caramelli/6018
 
 ## Timestamps
-- 0:00 Start of event
-- x:xx 
-- x:xx
-
-## Note: help us add timestamps here
-https://github.com/pymc-devs/video-timestamps
+- 0:00 Introduction Dario and Hessam
+- 0:26 Digitizing chemistry
```

Comment:
```suggestion
00:00 Introduction Dario and Hessam
00:26 Digitizing chemistry
```

Reply:
@SangamSwadiK Can you update the formatting for rest of time stamps as I have done?
- remove the first "-"
- use format `00:00`